Expose get_num_threads via pybind

kimishpatel opened the pull request:
Summary:
Main objective is to allow querying number of threads being used from python.

Later diff uses this mainly for testing.

Reviewed By: larryliu0820

Differential Revision: D87510750

larryliu0820 reviewed (approved):
Review automatically exported from Phabricator review in Meta.

kimishpatel committed:
Expose get_num_threads via pybind (#15944)

Summary:

Main objective is to allow querying number of threads being used from python.

Later diff uses this mainly for testing.

Reviewed By: larryliu0820

Differential Revision: D87510750
